lib/packets/PacketElements/Component.ts

```diff
@@ -130,12 +130,14 @@ SYSTEM_HANDLER.set(SystemType.Communications, {
 	check: (curr: CommsSystem, old: CommsSystem) => {
 		if("IsSabotaged" in curr && "IsSabotaged" in old) {
 			// assume SimpleComsSystem
-			return curr.IsSabotaged == curr.IsSabotaged
+			return curr.IsSabotaged != curr.IsSabotaged
```

Comment:
Is one of these supposed to be `old`?

Reply:
Yep, missed that one when copying into GitHub